tests/unit/plugins/inventory/test_kubevirt_inventory_options.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Red Hat, Inc.
+# Apache License 2.0 (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.kubevirt.core.plugins.inventory.kubevirt import (
+    InventoryOptions,
+)
+
+
+def test_inventory_options_defaults():
+    opts = InventoryOptions()
+    assert opts.api_version == "kubevirt.io/v1"
+    assert opts.kube_secondary_dns is False
+    assert opts.use_service is True
+    assert opts.create_groups is False
+    assert opts.append_base_domain is False
+    assert opts.host_format == "{namespace}-{name}"
```

Comment:
Shouldn't we also assert if fields `label_selector`, `network_name` and `base_domain` aren't set?

Reply:
I'll update that.